packages/common/src/hardhat/tasks/compile.js

```diff
@@ -1,14 +1,10 @@
 const { getAllFilesInPath } = require("@uma/common");
 
-const path = require("path");
```

Comment:
this is even simpler now. simply looks for matches on the array of whitelisted contract names.